crates/rspack_plugin_asset/src/lib.rs

```diff
@@ -420,6 +421,21 @@ impl ParserAndGenerator for AssetParserAndGenerator {
 
     result
   }
+  fn store(&self, extra_data: &mut HashMap<BuildExtraDataType, AlignedVec>) -> () {
+    extra_data.insert(
+      BuildExtraDataType::AssetParserAndGenerator,
+      to_bytes::<_, 256>(&self.parsed_asset_config).expect("Failed to store extra data"),
+    );
+  }
+
+  fn resume(&mut self, extra_data: &HashMap<BuildExtraDataType, AlignedVec>) -> () {
+    if let Some(data) = extra_data.get(&BuildExtraDataType::AssetParserAndGenerator) {
+      self.parsed_asset_config = unsafe {
```

Comment:
I believe there's a safe alternative here. Would you please change it to safe API?

Reply:
ok